packages/backend/src/Controllers/Election/getElectionResultsController.ts

```diff
@@ -31,17 +31,37 @@ const getElectionResults = async (req: IElectionRequest, res: Response, next: Ne
     const election = req.election
     let results: ElectionResults[] = []
     for (let race_index = 0; race_index < election.races.length; race_index++) {
-        const candidateNames = election.races[race_index].candidates.map((Candidate: any) => (Candidate.candidate_name))
-        const race_id = election.races[race_index].race_id
+        const race = election.races[race_index]
+        const candidateNames = race.candidates.map((Candidate: any) => (Candidate.candidate_name))
+        const candidateIDs = race.candidates.map((Candidate: any) => (Candidate.candidate_id))
+        const useWriteIns = race.enable_write_in && race.write_in_candidates && race.write_in_candidates.length>0
+        const writeInCandidates = useWriteIns && race.write_in_candidates ? race.write_in_candidates : [] 
```

Comment:
nit, I think you can avoid the terinary expression by doing ```race.write_in_candidates ?? []``

Reply:
I'll give that a shot but I may have tried that. Typescript wasn't very cooperative in this section.